platform/platform-app/src/main/resources/features.xml

```diff
@@ -1030,6 +1030,12 @@
         <bundle>mvn:ddf.platform.error/platform-error-impl/${project.version}</bundle>
     </feature>
 
+    <feature name="io" install="manual" version="${project.version}"
+             description="Internal tools for general purpose IO functions">
```

Comment:
:question: are these really general tools for IO functions or are they really specific to reading and writing config admin config files?